Resources/Locale/en-US/game-ticking/game-presets/preset-thief.ftl

```diff
@@ -0,0 +1,20 @@
+thief-role-greeting-human = 
+    You are a criminal scum.
+    You want to add to your collection
+    some property of Nanotrasen station. 
+    You're not a murderer. 
+    You are forbidden to kill.
```

Comment:
Line breaks are made on purpose, since the briefing system does not automatically break lines, and the text becomes very long.  The same was done for other antagonists.